gef.py

```diff
@@ -1356,7 +1370,8 @@ def cs_insn_to_gef_insn(cs_insn):
         sym_info = gdb_get_location_from_symbol(cs_insn.address)
         loc = "<{}+{}>".format(*sym_info) if sym_info else ""
         ops = [] + cs_insn.op_str.split(", ")
-        return Instruction(cs_insn.address, loc, cs_insn.mnemonic, ops)
+        opcodes = "".join("{:02x}".format(b) for b in cs_insn.bytes)
```

Comment:
it probably makes more sense for the opcodes that `Instruction` keep to be raw bytes, instead of a formatted string. It can format the bytes when `__format__` or `__str__` are called.